src/controllers/schemas.ts

```diff
@@ -280,14 +280,16 @@ export const transfersByResourceBetweenDomainsSchema = {
 }
 
 export const transfersByDomainSchema = {
-  summary: "Get all transfers with a specific domain as source or destination",
+  summary:
+    "Get all transfers with a specific domain as source or destination. Possible to get transfers by domain only as source and only as destination.",
   querystring: {
     type: "object",
     properties: {
       ...paginationSchema,
       domain: {
         type: "string",
-        default: "source",
+        default: undefined,
```

Comment:
This might break something on explorer-ui if it assumed source right now. 
@tcar121293 Do you know how this is used right now?

Reply:
@wainola can you validate this also

Reply:
this means it can be undefined, doesn't it? if so, yes, it would break explorer as is a property we are using to render info